### PR TITLE
add title sheet to documents

### PR DESCRIPTION
**Pull request by joka**
Todo: 

update frontend code

### CODE REVIEW

**Comment by xi** [comment]
For reference: This is about finally being able to sort `IDocumentVersion` listings. It is based on prior work in #1328 and #1342.

File: src/adhocracy_core/adhocracy_core/sheets/document.py [hunk 1]
```diff
@@ -43,7 +42,6 @@ class DocumentSchema(colander.MappingSchema):
     `elements`: structural subelements like sections
     """
 
-    title = SingleLine()
```
Comment (pallix) [inline src/adhocracy_core/adhocracy_core/sheets/document.py]: Docu of `DocumentSchema` needs to be updated

**Comment by pallix** [comment]
LGTM except the comment above.

**Comment by pallix** [comment]
@MagdaN you can review frontend part and merge.

**Comment by nidico** [comment]
LGTM.

**Pull request merged**